requirements/development.txt

```diff
@@ -276,7 +276,7 @@ pytest-forked==1.4.0
     #   pytest-xdist
 pytest-logging==2015.11.4
     # via -r requirements/base.txt
-pytest-mock==3.9.0
+pytest-mock==3.10.0
```

Comment:
No breaking change in the new version: https://github.com/pytest-dev/pytest-mock/blob/main/CHANGELOG.rst#3100-2022-10-05